### PR TITLE
Improve the deprecation message for using subclass of Rails::Application to start the Rails server

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**

### CODE REVIEW

File: railties/CHANGELOG.md [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
     *bogdanvlviv*
 
-*   Deprecate support of use `Rails::Application` subclass to start Rails server.
+*   Deprecate support for using `Rails::Application` subclass to start Rails server.
```
Comment (georgeclaghorn) [inline railties/CHANGELOG.md]: How about “for using *a* `Rails::Application` subclass”?

Comment (prathamesh-sonpatki) [reply railties/CHANGELOG.md]: @georgeclaghorn Done!

**Comment by eileencodes** [comment]
I don't know what is going on with CI here and even though I don't think this is related to your change it's too red to merge. Can you rebase with master and force push?

**Comment by prathamesh-sonpatki** [comment]
Rebased with latest master. The failure might be related to https://github.com/travis-ci/travis-ci/labels/2017-Q4-trusty-env-update

**Comment by prathamesh-sonpatki** [comment]
Tests are passing now

**Pull request merged**